source/nodejs/adaptivecards-site/themes/adaptivecards/source/css/style.css

```diff
@@ -724,23 +724,3 @@ input.bigbox {
   transform: scale(2.5);
   transform-origin: left;
 }
-
```

Comment:
Should we have this change and the serialization change in the main branch as well?

Reply:
I'm confused; that's what this PR is about?

Reply:
Yup 😆 I thought you were merging into the release branch - time for the weekend